reddit_liveupdate/templates/liveupdatehappeningnowbar.html

```diff
@@ -0,0 +1,22 @@
+<%!
+    from r2.lib.template_helpers import format_number
+%>
+
+<a href="/live/${thing.event._id}" class="liveupdate-happening-now">
+  <div class="rank-spacer"></div>
+  <div class="midcol-spacer"></div>
+  <span class="happening-now-logo"></span>
+
+  <div id="liveupdate-statusbar" class="status-bar live happening-now">
+    <p class="state">${_('LIVE')}</p>
+    <p role="presentation">&nbsp;-&nbsp;</p>
+    <p class="description">${thing.event.title}</p>
+    <p class="watching">
+    <%
+      num_watching = thing.event.active_visitors - (thing.event.active_visitors % 100)
```

Comment:
Are we doing this to avoid having mismatched numbers between this and the actual thread due to caching?

FYI, there is [a fuzzed boolean](https://github.com/reddit/reddit-plugin-liveupdate/blob/master/reddit_liveupdate/models.py#L39) that [gets used](https://github.com/reddit/reddit-plugin-liveupdate/blob/master/reddit_liveupdate/public/static/js/liveupdate/statusBar.js#L121) to either put in a tilde or not.